src/scicookie/{{cookiecutter.project_slug}}/build-system/base/dev-deps-config-pyproject.toml

```diff
@@ -48,7 +48,7 @@ sort_by_size = true
 verbose = false
 {% endif -%}
 
-{%- if cookiecutter.use_ruff == "yes" %}
+{%- if cookiecutter.use_ruff_linter == "yes" or cookiecutter.use_ruff_formatter == "yes" %}
```

Comment:
you can add at the end of the tool.ruff tables/sections

https://github.com/arxlang/astx/blob/main/pyproject.toml#L100-L101

```toml

[tool.ruff.format]
quote-style = "double"

```

Reply:
tried both `single` and `double`, still getting the same error.

Reply:
Hi @xmnlab, since the "double" format did not work, maybe we would change the generation of
`return '0.1.0'  # semantic-release` to use double quotes instead?